compiler/testSuite/14-array/fixedlength2-e.bal

```diff
@@ -0,0 +1,4 @@
+public function main() {
+    int[4] v = [1, 2, 3, 4];
+    v[4] = 4; // @error
```

Comment:
Error message is `error: value assigned to member of list is not a subtype of array member type`